functions/src/services/credential/credential.ts

```diff
@@ -112,33 +116,39 @@ export class Credential {
     )
   }
 
+  disabledError(): https.HttpsError {
+    return new https.HttpsError('permission-denied', 'User is disabled.')
+  }
```

Comment:
TypeScript can infer return value automatically

```suggestion
  disabledError() {
    return new https.HttpsError('permission-denied', 'User is disabled.')
  }
```